src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml

```diff
@@ -34,8 +34,9 @@
                 <Setter.Value>
                     <ControlTemplate TargetType="{x:Type ContextMenu}">
                         <StackPanel Margin="8,0,8,8"
-                                    Width="250">
-                            <ui:InCanvasSearchControl DataContext="{Binding InCanvasSearchViewModel}" />
+                                    Name="ContextMenuPanel">
+                            <ui:InCanvasSearchControl DataContext="{Binding InCanvasSearchViewModel}"
+                                                      Width="{Binding ElementName=ContextMenuPanel, Path=ActualWidth}" />
```

Comment:
After you have changed this, I realized the `StackPanel` is the one immediately above the `InCanvasSearchControl` (see, that's a good thing with named elements). Then my question becomes... can we not use a `Grid` (which automatically sizes all the children)? There might be a limitation why we can't use `Grid`, so I'm just wondering.